src/routes/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { RouteObject } from 'react-router-dom';
+
+import Course from '../modules/Course/routes';
+import Dashboard from '../modules/Dashboard/routes';
+
+// LOAD THE MODULE ROUTES HERE
+const routes: RouteObject[] = [...Dashboard, ...Course];
```

Comment:
The external layout should be kept in the main route, the main route builds the style around the container, which is the responsibility of this component. 
Nested routes are children of the main route and will only control and be responsible for dealing with the style and internal behavior of the containers.